app/javascript/.storybook/main.js

```diff
@@ -31,8 +31,6 @@ module.exports = {
       extensions: [...config.resolve.extensions, '.scss'],
       alias: {
         ...config.resolve.alias,
-        react: 'preact-compat',
```

Comment:
This is no longer required as `preact-compat` is built-in to the core of Preact X now.